dnf-behave-tests/dnf/plugins-core/download-debuginfo.feature

```diff
@@ -12,25 +11,27 @@ Scenario: Download a debuginfo for an RPM that doesn't exist
   Given I use repository "dnf-ci-fedora" as http
    When I execute dnf with args "download --debuginfo does-not-exist"
    Then the exit code is 1
-    And stderr contains "No package does-not-exist available"
+    And stderr contains "No package"
+    And stderr contains "does-not-exist"
```

Comment:
It would be better to test the output precisely. You can escape the quotation marks like this: `And stderr contains "No package \"does-not-exist\" available"`. There are more instances of this below.